### DISCUSSION TITLE
Shared account prevention (limit simultaneous login)

### DISCUSSION THREAD

**Discussion by jalberto** [zitadel/zitadel#7546] [Q&A]
Does zitadel include mitigation or configuration to prevent the same account to be used for multiple people?

**Comment by mffap** [comment]
You can [enforce multi-factor authentication](https://zitadel.com/docs/guides/manage/console/default-settings#login-behavior-and-access) and use [passkeys](https://zitadel.com/docs/concepts/features/passkeys). Enforcing [MFA](https://zitadel.com/docs/guides/manage/console/default-settings#multifactor-mfa) will require users to provide username/password and a second factor. Passkeys binds the authentication to a private key that is stored on a user's device. Of course you can also integrate with any third party identity provider (SSO/Identity brokering), that would create also a shared responsibility on that topic.

**Reply by jalberto** [reply]
I am thinking more in a desktop webapp, not mobile, so add MFA there can be too much friction.

Is zitadel aware of active sessions?

It doesn't need to be very accurate, it's OK if there is a small period of time when multiple sessions are active at once, but ideally at some point, the oldest session should close
